lib/handler/mruby/channel.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2015-2016 DeNA Co., Ltd., Kazuho Oku
```

Comment:
I appreciate your kindness.
I fixed it here: https://github.com/h2o/h2o/pull/1336/commits/60f1d0eaf56ed62a477a1a5be85f70e25798ea3d